### ISSUE TITLE
Error not being thrown when statement cancelled due to timeout

### ISSUE THREAD

**Issue by 0x-jj** [vitaly-t/pg-promise#867]
### Expected behavior

An error is thrown when a statement is cancelled due to a timeout

### Actual behavior

No error is thrown

### Steps to reproduce
```
try {
await idb.tx((t) => {
      const lockTimeout = t.none("SET LOCAL lock_timeout = '2s'");
      const q = t.none(query); // This causes an ERROR to be logged by the db and the statement is cancelled due to the lock timeout
      return t.batch([lockTimeout, q]);
    });
} catch (err) {
// No error is thrown
console.log(err)
}
```

### Environment

* Version of pg-promise: ^11.2.0
* OS type (Linux/Windows/Mac): Linux, aws lambda
* Version of Node.js: 16

**Comment by vitaly-t** [comment]
If you're not getting an error, it means that the [underlying driver](https://github.com/brianc/node-postgres) doesn't throw one. Whether it is a bug in the driver or something else, it is a little out of scope here. Check with the driver, and open a ticket there.

**Comment by vitaly-t** [comment]
[This addition](https://github.com/brianc/node-postgres/pull/2779) made it into the latest version. So if you are really using 11.2 (i'd check it), then it should work, somehow.

**Comment by 0x-jj** [comment]
> [This addition](https://github.com/brianc/node-postgres/pull/2779) made it into the latest version. So if you are really using 11.2 (i'd check it), then it should work, somehow. Or maybe it is a separate feature, I'm not sure, you'll need to research.

Interesting

I'm definitely on 11.2, though I don't see the lock_timeout supported when constructing the pgpromise config

Typescript says `Object literal may only specify known properties, and 'lock_timeout' does not exist in type 'IConnectionParameters<IClient>`

Is this just a typing thing and you expect it to still be passed down to the driver?

```
import PgPromise from 'pg-promise';

const pgp = PgPromise();

export const db = pgp({
  connectionString: config.databaseUrl,
  keepAlive: true,
  max: 10,
  connectionTimeoutMillis: 30 * 1000,
  query_timeout: 5 * 60 * 1000,
  allowExitOnIdle: true,
  lock_timeout: 5 * 60 * 1000, // has typescript error
});
```

**Comment by vitaly-t** [comment]
> Is this just a typing thing and you expect it to still be passed down to the driver?

It was indeed an omission, patched in [v11.3.0](https://github.com/vitaly-t/pg-promise/releases/tag/11.3.0).

**Comment by 0x-jj** [comment]
> > Is this just a typing thing and you expect it to still be passed down to the driver?
> 
> It was indeed an omission in TypeScript declaration, patched in [v11.3.0](https://github.com/vitaly-t/pg-promise/releases/tag/11.3.0).

Great, thanks!
